aws/eks/main.tf

```diff
@@ -104,7 +104,7 @@ resource "aws_cloudformation_stack" "nodes" {
     NodeAutoScalingGroupDesiredCapacity = "${var.capacity_desired}"
     NodeAutoScalingGroupMaxSize         = "${var.capacity_max}"
     NodeAutoScalingGroupMinSize         = "${var.capacity_min}"
-    NodeGroupName                       = "${var.name}"
+    NodeGroupName                       = "default"
```

Comment:
can we just make it a separate var? I can see use cases down the line where we need different sized nodes in a cluster and want to clarify naming.

Reply:
If it’s okay with you, I’d rather make it a variable if/when it becomes one, rather than pre-emptively doing so! 😄I definitely see the use case you mention as a possibility, but we don’t have any such plans at the moment, and I don’t know when we might get there!

Reply:
Feels like doing work twice, but I hear you.